src/index.js

```diff
@@ -95,9 +95,9 @@ function replaceGithubLink(message, markdownLink) {
  * @returns {string}
  */
 function escapeChars(message) {
-  message = message.replaceAll('<', '&lt;');
-  message = message.replaceAll('>', '&gt;');
-  message = message.replaceAll('&', '&amp;');
+  message = message.replace(/</g, '&lt;');
+  message = message.replace(/>/g, '&gt;');
+  message = message.replace(/&/g, '&amp;');
```

Comment:
@talyguryn  I change this because replaceAll giving throwing an error

Reply:
does it work correctly for multiple entities?

Reply:
Yup